### PR TITLE
fix: torch.dropout与 paddle.nn.functional.dropout的映射关系有问题

### LINKED ISSUE #7328: torch.dropout与 paddle.nn.functional.dropout的映射关系有问题
### 文档链接&描述 Document Links & Description

docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.dropout.md

torch.dropout：内建函数（是 TorchScript 在编译阶段可直接使用的低级 API，通常不是在 torch.* Python 层显式调用的）不是公开 Python API，而是一个内置的 C++ 算子。torch.dropout(input: Tensor, p: float, train: bool) -> Tensor

paddle.nn.functional.dropout：普通函数
https://github.com/PaddlePaddle/docs/blob/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.dropout.md
torch.nn.functional.dropout：普通函数

### 请提出你的建议/提个PR修复这个问题 Please give your suggestion/update a PR to fix it

建议直接删掉docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.dropout.md

### PR DESCRIPTION
**Pull request by JiwaniZakir**
Fixes #7328

The mapping document `torch.dropout.md` incorrectly classified `torch.dropout` and `paddle.nn.functional.dropout` as having only parameter name differences, when the two APIs have broader functional and behavioral differences that cannot be accurately described by this simplified mapping. Removes the incorrect file from `docs/guides/model_convert/convert_from_pytorch/api_difference/args_name_diff/` to prevent users from following misleading migration guidance.

### CODE REVIEW

**Comment by luotao1** [comment]
Please sign the CLA.

**Comment by zhwesky2010** [comment]
无需修改，目前`torch.dropout`与`paddle.nn.functional.dropout`两者用法存在不一致，需保留映射文档。可尝试对齐两者用法后，再删除文档。
